docs/table/construct_table.rst

```diff
@@ -750,7 +750,7 @@ The column data values, shape, and data type are specified in one of two ways:
   Examples include:
 
   - Python non-string type (float, int, bool)
-  - Numpy non-string type (e.g. np.float32, np.int64, np.bool)
+  - Numpy non-string type (e.g. np.float32, np.int64)
```

Comment:
I wasn't sure if `np.bool\_` would fix the doc problem, so I've removed it. Not sure if that's fine, but plain `bool` is mentioned in the line above.